opencontrail/map.jinja

```diff
@@ -1,96 +1,207 @@
+    services:
+        ['supervisor-config']
+control:
+  Debian:
+    pkgs:
+        ['contrail-control','contrail-dns']
+    services:
+        ['contrail-control', 'supervisor-control']
+  RedHat:
+    pkgs:
+         ['contrail-openstack-control']
+    services:
+         ['contrail-control', 'supervisor-control']
+database:
+  Debian:
+    pkgs:
+         ['cassandra', 'zookeeper', 'supervisor', 'openjdk-7-jre-headless','contrail-database']
+    cassandra_config: '/etc/cassandra/'
+    services:
+         ['supervisord-contrail-database','zookeeper']
+  RedHat:
+    pkgs:
+         ['contrail-openstack-database', 'zookeeper', 'supervisor', 'java-1.7.0-openjdk-headless']
+    cassandra_config: '/etc/cassandra/conf/'
+    services:
+         ['supervisord-contrail-database','zookeeper']
+web:
+  Debian:
+    pkgs:
+         ['contrail-web-controller', 'nodejs-legacy', 'gettext-base']
+    services:
+         ['contrail-webui-webserver','contrail-webui-jobserver']
+  RedHat:
+    pkgs:
+         ['contrail-web-controller']
+    services:
+         ['supervisor-webui']
+client:
+  Debian:
+    pkgs:
+         ['python-contrail']
+  RedHat:
+    pkgs:
+         []
+tor:
+  Debian:
+    agents: 1
+    bind:
+      port: 8086
 
-{% set collector = salt['grains.filter_by']({
-    'Debian': {
-        'pkgs': ['contrail-analytics', 'python-cassandra'],
-        'redis_config': '/etc/redis/redis.conf',
-        'services': ['supervisor-analytics','redis-server']
-    },
-    'RedHat': {
-        'pkgs': ['contrail-analytics'],
-        'redis_config': '/etc/redis.conf',
-        'services': ['supervisor-analytics','redis']
-    },
-}, merge=pillar.opencontrail.get('collector', {})) %}
+{%- elif vendor == 'juniper' -%}
 
-{% set compute = salt['grains.filter_by']({
-    'Debian': {
-        'pkgs': ['contrail-vrouter','contrail-vrouter-dkms', 'iproute2', 'haproxy'],
-        'services': ['supervisor-vrouter']
-    },
-    'RedHat': {
-        'pkgs': ['contrail-openstack-vrouter', 'haproxy', 'contrail-vrouter-source'],
-        'services': ['supervisor-vrouter']
-    },
-}, merge=pillar.opencontrail.get('compute', {})) %}
+common:
+  Debian:
+    vendor: juniper
+    source: {'engine': 'pkg', 'address': 'http://'}
+    pkgs:
+         ['dpkg-dev','contrail-utils', 'contrail-nodemgr', 'gettext-base', 'python-contrail']
+  RedHat:
+    vendor: juniper
+    source: {'engine': 'pkg', 'address': 'http://'}
+    pkgs:
+         ['contrail-utils', 'contrail-nodemgr']
+collector:
+  Debian:
+    pkgs:
+         ['contrail-analytics', 'python-cassandra-driver']
+    redis_config: '/etc/redis/redis.conf'
+    services:
+        ['supervisor-analytics','redis-server']
+  RedHat:
+    pkgs:
+         ['contrail-analytics', 'python-cassandra-driver']
+    redis_config: '/etc/redis.conf'
+    services:
+        ['supervisor-analytics','redis']
+compute:
+  Debian:
+    pkgs:
+         ['contrail-vrouter-dkms', 'iproute2', 'haproxy','contrail-vrouter-init']
```

Comment:
contrail-vrouter-init is missing in latest formula, was it removed, any idea if needed with juniper packaging?